cares_reinforcement_learning/util/plotter.py

```diff
@@ -51,8 +51,11 @@ def plot_data(
     )
 
     plt.legend(fontsize="15", loc="upper left")
-    fig_manager = plt.get_current_fig_manager()
-    fig_manager.window.setGeometry(100, 100, 800, 650)
+
+    # Resize the plotted figure. Pixel = DPI * Inch.
+    fig = plt.gcf()
```

Comment:
we don't want to be setting in inches - we want to be setting in pixels.

Reply:
Changed